tests/output/ConsoleInteractionTest.py

```diff
@@ -468,11 +453,17 @@ def test_print_results_project_wide(self):
                           {},
                           color=False)
             self.assertEqual(
-                "\n{}\n|    | [NORMAL] origin:\n|    | message"
-                "\n".format(STR_PROJECT_WIDE),
-                stdout.getvalue())
+                    "\n{0}\n|    | [NORMAL] origin:\n|    | {1}""".format(
+                             STR_PROJECT_WIDE,
+                             highlight("message", TextLexer(),
+                                       TerminalTrueColorFormatter(
+                                            style=BackgroundMessage))),
+                            stdout.getvalue())
 
     def test_print_results_for_file(self):
+        def highlightoutput(text, lexer=self.lexer):
+            return highlight(text, lexer, TerminalTrueColorFormatter(
+                style=BackgroundSourceRange)).rstrip("\n")
```

Comment:
you should also introduce a `style` parameter with a default, then you can use this function on even more places :+1: